crowdin.yaml

```diff
@@ -1,19 +1,3 @@
-project_identifier: django-girls-tutorial
-api_key: 2be4b1707d0745bc96e9a5c9ffb4ca9b
```

Comment:
Need to be removed and if not changed before, `api_key` need to be revoked/changed.

See: https://support.crowdin.com/api/api-integration-setup/